### PR TITLE
[WPF] Fix FormsPanel MeasureOverride negative requested size

### PR DESCRIPTION
**Pull request by tessarolli**
### Description of Change ###

On WPF Platform, Element.Measure().Request(); sometimes return negative values (-1) on Width or Height. If that happened, instantiating a new Windows.Size with negative values would throw an exception that prevented software execution.
This fix is to prevent that from happening, as it checks and replaces -1 values with zero.

### Issues Resolved ### 

Fix Negative Width or Height on new Windows.Size();

### API Changes ###
 None

### Platforms Affected ### 

WPF

### Behavioral/Visual Changes ###

None

### Before/After Screenshots ### 

Not applicable

### PR Checklist ###
- [x] Targets the correct branch

### CODE REVIEW

File: Xamarin.Forms.Platform.WPF/FormsPanel.cs [hunk 1]
```diff
@@ -82,6 +82,13 @@ protected override System.Windows.Size MeasureOverride(System.Windows.Size avail
 			if (double.IsInfinity(elementDesiredWidth) || double.IsPositiveInfinity(elementDesiredHeight))
 			{
 				Size request = Element.Measure(elementDesiredWidth, elementDesiredHeight, MeasureFlags.IncludeMargins).Request;
+
+				if (request.Width == -1)
```
Comment (VladislavAntonyuk) [inline Xamarin.Forms.Platform.WPF/FormsPanel.cs:86]: I think it would be better to check if request.Width < 0. it will cover all invalid values

**Comment by VladislavAntonyuk** [comment]
Thank you for the PR. please make sure you didn't break fixes for: 
#6436
#8039
#9029
#5682
#6842

File: Xamarin.Forms.Platform.WPF/FormsPanel.cs (hunk 1 above)
Comment (tessarolli) [reply Xamarin.Forms.Platform.WPF/FormsPanel.cs]: OK, did it

**Comment by tessarolli** [comment]
Hi Vladislav,

I've update to check for all negative values.

Personally, I dont believe it will break any of those fixes you mentioned, because in case you do receive a negative request size, it would throw an exception trying to instantiate a new Size(), that would prevent software execution.

I did some testings on my end and all seems fine.

**Review by samhouts — APPROVED** [review]

**Pull request merged**